Post install hooks need to call depmod to properly load kernel modules after
reboot.

diff --git a/cumulus/etc/kernel/postinst.d/update-cumulus b/cumulus/etc/kernel/postinst.d/update-cumulus
index 0333980..cd8d311 100755
--- a/cumulus/etc/kernel/postinst.d/update-cumulus
+++ b/cumulus/etc/kernel/postinst.d/update-cumulus
@@ -50,3 +50,5 @@ $cmd
 cmd="mv -f $initrd_newimg_src $initrd_newimg_dst"
 echo "$cmd"
 $cmd
+
+depmod -a $version
diff --git a/cumulus/postinst.linux-image.powerpc b/cumulus/postinst.linux-image.powerpc
index 2aabfea..fb775d4 100755
--- a/cumulus/postinst.linux-image.powerpc
+++ b/cumulus/postinst.linux-image.powerpc
@@ -1,5 +1,5 @@
 #!/bin/sh
-# postinst script for switchd
+# postinst script for kernel
 #
 # see: dh_installdeb(1)
 
@@ -54,4 +54,7 @@ sh $install_file $kimage_path $slot || {
 
 cd /boot && ln -sf $kimage_name $kimage_linkname
 
+echo "depmod -a =V"
+depmod -a =V
+
 exit 0
